test/smoke/src/sql/testConfig.ts

```diff
@@ -78,9 +114,19 @@ function getEnumMappingEntry(mapping: any, enumValue: any): INameDisplayNamePair
 	}
 }
 
-export async function getDefaultTestingServer(): Promise<TestServerProfile> {
+export async function getAzureServer(): Promise<TestServerProfile> {
+	let servers = await getTestingServers();
+	return servers.filter(s => s.version === '2012')[0];
```

Comment:
why is the version 2012?

Reply:
Because Azure is always 2012. I still think we should stop using the years as keys though...as suggested before using something like an EngineType enum would be **much** clearer.

Reply:
+1